packages/devtools_app/test/flutter/network_profiler_test.dart

```diff
@@ -0,0 +1,302 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
```

Comment:
does this test have to run on the vm?

Reply:
Yes, as we're currently using `dart:io` to load test data.

Reply:
Yes, this test loads data from disk using dart:io.